integration-test/docker-compose_template.yml

```diff
@@ -81,6 +81,8 @@ services:
       - INTEGRATIONTEST_AWS_CLOUDSTORAGE_S3_INSTANCEPROFILE
       - INTEGRATIONTEST_AWS_CLOUDSTORAGE_BASELOCATION
       - INTEGRATIONTEST_AWS_CLOUDSTORAGE_FILESYSTEMTYPE
+      - INTEGRATIONTEST_AWS_DISKENCRYPTION_ENVIRONMENTKEY
+      - INTEGRATIONTEST_AWS_DISKENCRYPTION_DATAHUBKEY
```

Comment:
What about the analogous variables for the Azure encryption keys?

Reply:
Oh, okay, I did not notice this was not needed for the regular execution of Azure E2E test. :smile: